.github/workflows/main.yml

```
@@ -1,65 +1,110 @@
-# Secrets: ITCH_IO_API_KEY
-
-name: "Godot CI/CD on push to main"
+name: "Godot CI/CD"
 
 env:
-  GODOT_VERSION: 4.4
+  GODOT_VERSION: 4.4.1
   PROJECT_NAME: 'ci_example'
 
 on:
   workflow_dispatch:
   push:
     branches:
       - main
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - ready_for_review
+      - review_requested
+      - review_request_removed
+      - converted_to_draft
```

We probably don't need to run this workflow on the following. I'm a bit iffy on `edited`. I'm not sure if it means the pull request got pushed to, or if the text of pull request was changed.

```suggestion
      - opened
      - edited
      - synchronize
      - reopened
```

---

Non-blocking ^

Just remove them and merge whenever you are ready

---

The [edited](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request) action of the pull_request event means that: The title or body of a pull request was edited, or the base branch of a pull request was changed.

Fixed in the latest commit.